daemon/graphdriver/copy/copy_test.go

```diff
@@ -45,6 +48,52 @@ func TestCopyWithoutRange(t *testing.T) {
 	doCopyTest(t, &copyWithFileRange, &copyWithFileClone)
 }
 
+func TestCopyDir(t *testing.T) {
+	srcDir := "testsrc"
```

Comment:
Having all these empty files checked in feels kind of awkward, can you just generate them at the start of the test in another temp dir?